vendors/espressif/boards/esp32/aws_tests/config_files/aws_wifi_config.h

```diff
@@ -59,7 +59,8 @@
 /**
  * @brief Max passphrase length
  */
-#define wificonfigMAX_PASSPHRASE_LEN          ( 64 )
+// Maximum allowed WPA2 passphrase length (per specification) is 63
```

Comment:
Ditto for this file. Please reformat this info into comment starting at Line#59 as suggested in my other comment.